cvprac/cvp_api.py

```diff
@@ -667,9 +687,14 @@ def add_configlet(self, name, config):
                        timeout=self.request_timeout)
 
         # Get the key for the configlet
-        data = self.clnt.get('/configlet/getConfigletByName.do?name=%s'
-                             % urllib.quote_plus(name),
-                             timeout=self.request_timeout)
+        try:
```

Comment:
I am going to think about this try/except format for a day. Functionality looks good but we may be able to handle this in a cleaner way.

Reply:
Agreed, there is likely a much cleaner way of performing this. I had considered created a new object and setting it to the method of urllib.quote_plus or urllib.parse.quote_plus based on the version of python the user is running.

looking forward to seeing what you come up with

Reply:
Can you try removing the try/except clauses in the methods and apply it during the imports?

try:
    from urllib import quote_plus as qplus
except AttributeError:
    from urllib.parse import quote_plus as qplus

And then in the methods you always reference the alias qplus.

Example:
return self.clnt.get('/configlet/getConfigletByName.do?name=%s'
                             % **qplus(name)**, timeout=self.request_timeout)